.github/workflows/build-wsl.yaml

```diff
@@ -252,6 +266,15 @@ jobs:
           name: build-artifacts-${{ matrix.WslID }}
           path: ${{ env.workDir }}/${{ env.buildInfoPath }}/${{ matrix.WslID }}-*
 
+      - name: Notify ISO Tracker
```

Comment:
Same, I would do that after updating the build artefacts.

Actually, thinking about it, we are only interested when we store the version in the "db" (yes, the wiki). So maybe another approach would be to do it in `update-build-artifacts` job rather, for all new application packages (==images) we uploaded to the store.